src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/directory.py

```diff
@@ -0,0 +1,13 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.profiles import get_sdk, ResourceType
+
+
+def list_share_directories(cmd, client, share_name, directory_name=None, timeout=None):
+    t_dir_properties = get_sdk(cmd.cli_ctx, ResourceType.DATA_STORAGE, 'file.models#DirectoryProperties')
```

Comment:
Again, here you should just be able to call `cmd.get_models('file.models#DirectoryProperties')`

Reply:
will the default `mod` parameter disrupt it?